api/src/controllers/correction-request.js

```diff
@@ -53,6 +53,12 @@ router.post("/:youngId", passport.authenticate("referent", { session: false, fai
       request.sentAt = sentAt;
       request.status = "SENT";
 
+      // If unreadable/incorrect/other issue with id proof, delete it.
+      if (request.field === "cniFile" && ["UNREADABLE", "OTHER", "NOT_SUITABLE"].includes(request.reason) && request.status === "SENT") {
```

Comment:
```suggestion
      if (request.field === "cniFile" && ["UNREADABLE", "OTHER", "NOT_SUITABLE"].includes(request.reason) {
```

T'as pas besoin du request.status === SENT, vu qu'on le set juste sur la ligne d'avant, ca sera toujours vrai.